src/unix/timer.c

```diff
@@ -90,6 +87,22 @@ int uv_timer_start(uv_timer_t* handle,
 }
 
 
+int uv_timer_init(uv_loop_t* loop, uv_timer_t* handle) {
+  uv__handle_init(loop, (uv_handle_t*)handle, UV_TIMER);
+  handle->timer_cb = NULL;
+  handle->repeat = 0;
+  return 0;
+}
+
+
+int uv_timer_start(uv_timer_t* handle,
+                   uv_timer_cb cb,
+                   uint64_t timeout,
+                   uint64_t repeat) {
+  return timer_start_clamped(handle, cb, get_clamped_timeout(handle->loop->time, timeout), repeat);
```

Comment:
ok, actually there are other codes longer than mine in this file so that I have thought that the long line is okay for the project.
I will fix it and rebase codes to latest head asap.